xml/ha_cluster_lvm.xml

```diff
@@ -856,13 +911,27 @@ vdb                     253:16   0   20G  0 disk
 └─cluster--vg2-test--lv 254:5    0   12G  0 lvm
 vdc                     253:32   0   20G  0 disk</screen>
    </step>
-   <step>
+   <step xml:id="step.data-offset">
     <para>
       Create a cluster MD device <filename>/dev/md0</filename> with the disk
      <filename>/dev/vdc</filename>:
     </para>
     <screen>&prompt.root;mdadm --create /dev/md0 --bitmap=clustered \
-  --metadata=1.2 --raid-devices=1 --force --level=mirror /dev/vdc</screen>
+    --metadata=1.2 --raid-devices=1 --force --level=mirror --data-offset=128 \
+    /dev/vdc</screen>
+    <para>
+    <!--
+     https://bugzilla.suse.com/show_bug.cgi?id=1123496
+    -->
+     The <option>--data-offset</option> option reduces unneeded padding space
+     from the array. The offset must be large enough for cluster MD to write its
+     data to it. We recommend 128 KB. On the other hand, the offset must be small
```

Comment:
Same here with `&nbsp;` between "128" and "KB".

Reply:
Done.